### PR TITLE
Enhancement/nopd seps

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: datavalid.yml [hunk 1]
```diff
@@ -60,6 +60,10 @@ schemas:
           - female
           - male
       - name: agency
+      - name: class_id
+      - name: assignment_id
```
Comment (pckhoi) [inline datavalid.yml:64]: Would be nice to add `description` to more obscured name like `class_id` and `assignment_id`.

Comment (ayyubibrahimi) [reply datavalid.yml:64]: Agreed. I have no idea what they actually mean, yet. Working on getting more information.

Comment (pckhoi) [reply datavalid.yml:64]: I think we should always discard columns that we don't understand. Data without meaning cannot serve any purpose. If the intention is to remind ourselves to find out more about some columns, then I think it's better to leave `# TODO:` comments in the clean scripts. Unless you have other reasons to leave them in.

Comment (ayyubibrahimi) [reply datavalid.yml:64]: My original thinking was that someone who is downloading these tables for research purposes may know what they mean, as they are likely to have more insight than us into NOPD. Happy to leave `# TODO:` comments in the clean scripts if you think that concern is unwarranted.

Comment (pckhoi) [reply datavalid.yml:64]: Yeah, it's just hard for me to imagine a case where someone who look at the data we produce and would know more than us. They will probably have to look at our scripts anyway and even the source data as well. And the web developers will have to modify their SQL schema to accommodate any new field. So it's probably more trouble than it's worth.

Comment (ayyubibrahimi) [reply datavalid.yml]: Makes sense to me. Removed the columns. This is ready for review.

**Comment by pckhoi** [comment]
I see some changes that should belong to the ocr branch. Does this branch share some history? I'll review again after OCR changes are merged.

**Comment by ayyubibrahimi** [comment]
@pckhoi Ready for review

File: dvc/data/raw/ner.dvc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+wdir: ../../../data/raw
+outs:
+- md5: 43d5c9ca303e4eccba73044b10eef279.dir
+  size: 262617
+  nfiles: 1
+  path: ner
```
Comment (pckhoi) [inline dvc/data/raw/ner.dvc:6]: This file seems redundant. Please remove it.

File: dvc/data/raw/post_officer_history.dvc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+wdir: ../../../data/raw
+outs:
+- md5: b8bc7e6f1d623ce5f582a781dcabded2.dir
+  size: 256033
+  nfiles: 1
+  path: post_officer_history
```
Comment (pckhoi) [inline dvc/data/raw/post_officer_history.dvc:6]: This file also seems redundant.

File: datavalid.yml [hunk 4]
```diff
@@ -60,6 +60,8 @@ schemas:
           - female
           - male
       - name: agency
+      - name: years_of_service
+      - name: rank_desc
```
Comment (pckhoi) [inline datavalid.yml:64]: I think you should put `years_of_service` in the `event` table instead. `rank_desc` might change for an officer so it should not be added to `personnel`.

File: datavalid.yml [hunk 5]
```diff
@@ -301,6 +303,7 @@ schemas:
         description: >
           further description of the award
       - name: left_reason
+      - name: left_reason_desc
```
Comment (pckhoi) [inline datavalid.yml:306]: What is the difference between `left_reason` and `left_reason_desc`? Would be nice if you leave a `description` for `left_reason_desc` at least.

Comment (ayyubibrahimi) [reply datavalid.yml:306]: Thanks.

**Comment by ayyubibrahimi** [comment]
Ready for review.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**